src/drivers/mct/bld/testdir/env_build.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<file id="env_build.xml">
+  <header>
+    These variables SHOULD NOT be changed once the model has been built.
+    urrently, these variables are not cached.
+    Note1: users SHOULD NOT modify BUILD_COMPETE below
+    this is done automatically by the scripts.
+    </header>
+  <group id="build_grid">
+    <entry id="GRID" value="a%1x1_tropicAtl_l%1x1_tropicAtl_oi%1x1_tropicAtl_r%null_m%reg_g%null_w%null">
+      <type>char</type>
+      <desc>Model grid - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="ATM_GRID" value="1x1_tropicAtl">
+      <type>char</type>
+      <desc>atmosphere grid - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="LND_GRID" value="1x1_tropicAtl">
+      <type>char</type>
+      <desc>land grid - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="OCN_GRID" value="1x1_tropicAtl">
+      <type>char</type>
+      <desc>ocn grid - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="ICE_GRID" value="1x1_tropicAtl">
+      <type>char</type>
+      <desc>ice grid (must equal ocn grid) - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="ROF_GRID" value="null">
+      <type>char</type>
+      <desc>river runoff (rof) grid</desc>
+    </entry>
+    <entry id="GLC_GRID" value="null">
+      <type>char</type>
+      <valid_values>gland20,gland10,gland5,gland5UM,gland4,null</valid_values>
+      <desc>glacier (glc) grid - DO NOT EDIT (for experts only)</desc>
+    </entry>
+    <entry id="WAV_GRID" value="null">
+      <type>char</type>
+      <desc>wave model (wav) grid</desc>
+    </entry>
+    <entry id="MASK_GRID" value="UNSET">
+      <type>char</type>
+      <desc>grid mask - DO NOT EDIT (for experts only)</desc>
+    </entry>
+  </group>
+  <group id="build_def">
+    <entry id="CALENDAR" value="NO_LEAP">
+      <type>char</type>
+      <valid_values>NO_LEAP,GREGORIAN</valid_values>
+      <desc>calendar type</desc>
+    </entry>
+  </group>
+  <group id="build_macros">
+    <entry id="MPILIB" value="mpich2">
+      <type>char</type>
+      <valid_values/>
+      <desc>mpi library (must match one of the supported libraries) -
+    ignored once Macros has been created
+    Set in $CIMEROOT/machines/config_machines.xml for each supported machine.
+    Must be explicitly set in env_build.xml for userdefined machine.</desc>
+    </entry>
+    <entry id="PIO_VERSION" value="1">
+      <type>integer</type>
+      <valid_values>1,2</valid_values>
+      <desc>PIO library version</desc>
+    </entry>
+
+  </group>
+</file>
```

Comment:
The above also has the wrong schema for an entry_id. Who is using this? Is it needed?